examples/twitter-connector/pom.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2022, salesforce.com, inc.
+  ~ All rights reserved.
+  ~ SPDX-License-Identifier: BSD-3-Clause
+  ~ For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+  -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.salesforce.einsteinbot</groupId>
+  <artifactId>einstein-bot-twitter-connector</artifactId>
+  <version>1.0.5-SNAPSHOT</version>
+  <name>einstein-bot-twitter-connector</name>
+
+  <properties>
+    <java.version>1.8</java.version>
+    <maven.compiler.source>1.8</maven.compiler.source>
+    <maven.compiler.target>1.8</maven.compiler.target>
+    <bootstrap.version>4.2.1</bootstrap.version>
+    <okhttp-version>4.9.2</okhttp-version>
+  </properties>
+
+  <parent>
+    <groupId>org.springframework.boot</groupId>
+    <artifactId>spring-boot-starter-parent</artifactId>
+    <version>2.5.8</version> <!-- lookup parent from repository -->
+  </parent>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.salesforce.einsteinbot</groupId>
+      <artifactId>einstein-bot-channel-connector-java-starter</artifactId>
+      <version>1.0.5-SNAPSHOT</version>
```

Comment:
Can we use the latest release version `1.0.4` ?

Reply:
That's what I had but changed it because `ui-connector` uses the `SNAPSHOT` version. I assumed it is because we want to upgrade and maintain these examples in sync with the current version of channel connector.

Reply:
oh ok. May after our initial development is done, we can change both to the release version.